VideoApp/VideoApp/NewCode/Screens/Room/RoomView.swift

```diff
@@ -101,21 +113,27 @@ struct RoomView: View {
 
 struct RoomView_Previews: PreviewProvider {
     static var previews: some View {
+        let roomName = "Demo"
```

Comment:
I found this made it clearer what each preview is for since string literals stand out so much.